source/main.py

```diff
@@ -1,3 +1,2 @@
 if __name__ == "__main__":
-    print('spam')
-    print('hoi dit is Jacob')
+    print('Please keep this space as clean as posible.')
```

Comment:
both are messy. This did not fix anything.
Instead look into how to keep things clean and not just say that they should